source/css/sass/patterns/molecules/teaser-list.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../variables/color";
+@import "../../mixins/decorations";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list__title {
+  @include block-spacing();
+  @include label-content-typography();
+  @include set-font-size-and-line-height(14px);
+  color: $color-text-normal;
```

Comment:
`color` can be passed as the first argument to the call to `@include label-content-typography()` on line 8; doing so means this line isn't needed.